src/main/java/org/folio/service/transformationfields/TransformationFieldsConfig.java

```diff
@@ -60,7 +60,11 @@
   PERMANENT_LOAN_TYPE_ID("permanentLoanTypeId", "$.item.permanentLoanTypeId", LOAN_TYPES),
   STATUS("status", "$.item.status.name"),
   VOLUME("volume", "$.item.volume"),
-  YEARCAPTION("yearCaption", "$.item.yearCaption");
+  YEARCAPTION("yearCaption", "$.item.yearCaption"),
+  CALL_NUMBER("callNumber", "$.item.effectiveCallNumberComponents.callNumber"),
+  CALL_NUMBER_PREFIX("callNumberPrefix", "$.item.effectiveCallNumberComponents.prefix"),
+  CALL_NUMBER_SUFFIX("callNumberSuffix", "$.item.effectiveCallNumberComponents.suffix"),
+  CALL_NUMBER_TYPE("callNumberType", "$.item.effectiveCallNumberComponents.typeId");
```

Comment:
This needs metadata to be sent right? Because we have to replace the UUID with corresponding value?

Reply:
As I understood from Magda's answer in jira call number type should be set as id to marc file. In this case, we don't need metadata and jsonPath is enough